src/libs/data-utils.js

```diff
@@ -380,11 +385,9 @@ export const EntityUploader = class EntityUploader extends Component {
               ...Utils.newTabLinkProps
             }, ['Learn more ', icon('pop-out', { size: 12 })])
           ]),
-          isInvalid && div({
-            style: { color: colors.danger(), fontWeight: 'bold', fontSize: 12, marginTop: '0.5rem' }
-          },
-          ['Invalid format: Data does not start with entity or membership definition.']
-          )
+          div({ style: errorTextStyle },
+            [(isInvalid && 'Invalid format: Data does not start with entity or membership definition.') ||
+            (showInvalidEntryMethodWarning && 'Invalid Data Entry Method: Copy and paste only')])
```

Comment:
If these two cases need to be mutually exclusive, it might be cleaner to use a `Utils.cond` here